apps/hash-api/src/graphql/type-defs/knowledge/entity.typedef.ts

```diff
@@ -43,7 +43,7 @@ export const entityTypedef = gql`
     """
     Get a subgraph rooted at all entities at their latest version.
```

Comment:
Likely not a high priority right now, but we should probably update the docs for these and the ones for ontology types

Reply:
[Update docs](https://github.com/hashintel/hash/pull/2125/commits/d909a3429cc06d46a218918df4e93b84cb461cfc)